contracts/core/extensions/CoreIssuanceOrder.sol

```diff
@@ -403,6 +403,9 @@ contract CoreIssuanceOrder is
             _order.makerAddress
         );
 
+        // Verify maker token used is less than amount allocated that user signed
+        // require(makerTokenAmountUsed <= requiredMakerTokenAmount);
```

Comment:
@bweick Is this necessary? If makerTokenAmountUsed is greater, than the issuance order taker actually gets less. LMK if i'm misunderstanding something